integrationtests/agent/bundle_deployment_status_test.go

```diff
@@ -0,0 +1,209 @@
+package agent
+
+import (
+	"time"
+
+	"github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+	"github.com/rancher/wrangler/pkg/genericcondition"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	bundle           = "bundle"
+	timeout          = 5 * time.Second
+	svcName          = "svc-test"
+	svcFinalizerName = "svc-finalizer"
+)
+
+var _ = Describe("BundleDeployment status", Ordered, func() {
+	When("New bundle deployment is created", func() {
+		BeforeAll(func() {
+			// this BundleDeployment will create a deployment with the resources from assets/deployment.yaml
```

Comment:
I don't understand this comment, the `createBundleDeploymentV1` doesn't seem to use any file from assets?
Also there is no deployment.yaml, only deployment-v1.yaml ;)

Reply:
createBundleDeploymentV1 has a DeploymentID = v1, and v1 will deploy the content from assets/deployment-v1.yaml